packer.mk

```diff
@@ -66,7 +66,10 @@ else
 endif
 
 build-packer-windows:
-ifeq (${MODE},sigMode)
+ifeq (${MODE},gen2Mode)
+	@echo "${MODE}: Building with Hyper-v generation 2 VM and save to Classic Storage Account"
+	@packer build -var-file=vhdbuilder/packer/settings.json vhdbuilder/packer/windows-vhd-builder-gen2.json
+else ifeq (${MODE},sigMode)
```

Comment:
gen2 also needs to support sigMode. sigMode: publish the built VHD as the sig image in the resource group and do not delete it in cleanup step. We can use this sig image for test later.

Reply:
Maybe add one branch `ifeq (${MODE},gen2SigMode)`? See the options above?

Reply:
Only two branches `sigMode` and `Default`. This file will check `GEN2_SIG_FOR_PRODUCTION` and would go into the real "`sigMode`" if this variable is `False`. (sig is for test purpose only)